src/ray/core_worker/task_manager.h

```diff
@@ -268,8 +268,9 @@ class TaskManager : public TaskFinisherInterface, public TaskResubmissionInterfa
   /// responsible for making sure that these dependencies become available, so
   /// that the resubmitted task can run. This is only populated if the task was
   /// not already pending and was successfully resubmitted.
-  /// \return OK if the task was successfully resubmitted or was
-  /// already pending, Invalid if the task spec is no longer present.
+  /// \return true if the task was successfully resubmitted (task or actor being
```

Comment:
@jjyao I actually have a question on how we retry and retrieve retry result, task manager only *submit* task, but not blocking wait their completion; task recovery manager also doesn't block wait. Is there any concern?

Reply:
Discussed offline. It doesn't block, it's async operation.